Gruntfile.js

```diff
@@ -259,7 +259,7 @@ module.exports = grunt => {
   grunt.registerTask('docs', ['docs-css', 'vendor']);
   grunt.registerTask('release', ['swatch', 'docs']);
 
-  grunt.registerTask('server', 'connect:keepalive');
+  grunt.registerTask('server', 'connect');
```

Comment:
I'm unsure if this task makes any sense/is used? Should it be an alias of `['connect', 'watch']`?

Reply:
Done